### PR TITLE
Try to auto-detect token type used in runtime transactions

### PR DESCRIPTION
**Pull request by csillag**
Use `tx.body` to figure out if it's something special, or the default native token.

### CODE REVIEW

**Review by lubej — APPROVED** [review]

File: src/oasis-nexus/api.ts [hunk 1]
```diff
@@ -752,7 +752,9 @@ export const useGetRuntimeEvents: typeof generated.useGetRuntimeEvents = (
                               event.body.amount.Amount,
                               paraTimesConfig[runtime].decimals,
                             ),
-                            Denomination: getTokensForScope({ network, layer: runtime })[0].ticker, // TODO find this out from event data
+                            Denomination:
+                              event.body?.Denomination ??
```
Comment (lukaw3d) [inline src/oasis-nexus/api.ts:756]: This doesn't exist. `event.body?.amount.Denomination` exists.
But `event.body.amount.Denomination === ''` above means that it is empty

Comment (csillag) [reply src/oasis-nexus/api.ts:756]: I have seen both existing on different data, IIRC.

Comment (csillag) [reply src/oasis-nexus/api.ts:756]: Well, maybe not.

Comment (csillag) [reply src/oasis-nexus/api.ts:756]: Right, when this code is executed, it's always empty. Removed useless condition in #1285.

**Pull request merged**